doc/tier2.md

```diff
@@ -1,15 +1,15 @@
-## Tier 2 Storage
+## LongTermStorage
```

Comment:
Please change the filename to "longtermstorage.md"

Reply:
Done